internal/kibana/connector_data_source_test.go

```diff
@@ -0,0 +1,45 @@
+package kibana_test
+
+import (
+	"testing"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceKibanaConnector(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.Providers,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceConnector,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.elasticstack_kibana_action_connector.myconnector", "name", "myconnector"),
+					resource.TestCheckResourceAttr("data.elasticstack_kibana_action_connector.myconnector", "space_id", "default"),
+					resource.TestCheckResourceAttr("data.elasticstack_kibana_action_connector.myconnector", "connector_type_id", ".slack"),
+					resource.TestCheckResourceAttrSet("data.elasticstack_kibana_action_connector.myconnector", "connector_id"),
+				),
+			},
+		},
+	})
+}
+
+const testAccDataSourceConnector = `
+provider "elasticstack" {
+  elasticsearch {}
+  kibana {}
+}
+
+resource "elasticstack_kibana_action_connector" "slack" {
+	name              = "myconnector"
+	connector_type_id = ".slack"
```

Comment:
Does it make sense to cover other connector types as well (similar to `internal/kibana/connector_test.go`)?

Reply:
E.g., I guess, we can amend `connector_test.go` to cover the new datasource.